1891059_안민재/Backjoon2573.java

```diff
@@ -0,0 +1,123 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.StringTokenizer;
+
+public class Backjoon2573 {
+    static int N,M;
+    static int[][] arr;
+    static boolean[][] visited;
+    static List<Node> list=new ArrayList<>();
+
+    static int[] px={-1,1,0,0};
+    static int[] py={0,0,-1,1};
```

Comment:
변하지 않는 값은 static final을 이용하는게 안전해보이네요. static final로 선언한 값은 보통 대문자로 작성합니다!
```suggestion
    static final int[] PY={0,0,-1,1};
```

Reply:
진짜 사랑스러운 리뷰네요 ㄳ